### DISCUSSION TITLE
Payments

### DISCUSSION THREAD

**Discussion by ralyodio** [owncast/owncast#1721] [Ideas] [answered]
I would like to have a paid subscriber plan.  Ie $10/session

without a way to monetize I think this solution isn’t work

**Comment by gabek** [comment, marked as answer]
It is up to you to determine if, and how you want to monetize. Owncast can power your video, but it is out of scope for Owncast to be your payment platform.

**Comment by ralyodio** [comment]
I’m just looking grid stripe integration or coin payments is there a way to weite a plug-in

**Reply by gabek** [reply]
Sure, you can extend Owncast to build anything you like. https://owncast.online/thirdparty/
